weave/type_serializers/Image/image_test.py

```diff
@@ -100,3 +105,47 @@ def test_image_as_call_io_refs(client: WeaveClient) -> None:
     assert image_as_solo_output_call.output.tobytes() == exp_bytes
     assert image_as_input_and_output_part_call.inputs["in_img"].tobytes() == exp_bytes
     assert image_as_input_and_output_part_call.output["out_img"].tobytes() == exp_bytes
+
+
+def test_image_from_path(client: WeaveClient) -> None:
+    with tempfile.TemporaryDirectory() as temp_dir:
+        path_str = os.path.join(temp_dir, "img.png")
+        img = Image.new("RGB", (512, 512), "purple")
+        img.save(path_str)
+        exp_bytes = img.tobytes()
+
+        @weave.op
+        def save_image(path: Path) -> dict:
+            return {"out-img-path": path}
+
+        save_image(Path(path_str))
+
+        call = save_image.calls()[0]
+        assert call.output["out-img-path"].tobytes() == exp_bytes
+
+        @weave.op
+        def save_stuff_with_image_embedded(path: Path) -> dict:
+            return {"out": f"Some random text + a path: {path}"}
+
+        save_stuff_with_image_embedded(Path(path_str))
+        call = save_stuff_with_image_embedded.calls()[0]
+        assert call.output["out"] == f"Some random text + a path: {path_str}"
+
+
+def test_image_from_remote_path(client: WeaveClient) -> None:
+    remote_path = "https://www.gstatic.com/webp/gallery/1.jpg"
```

Comment:
we could host this ourself but I think this image has been around since the dawn of the internet and probably isn't going anywhere anytime soon. I suspect that hosting an image we control might be more volatile.